packages/terra-core-docs/src/terra-dev-site/doc/action-header/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,233 @@
+import { Badge } from 'terra-action-header/package.json?dev-site-package';
+import { Notice } from '@cerner/terra-docs';
+import Whitespace from "../../common/layout-helpers/Whitespace";
+import ActionHeaderExample from "./example/BackActionHeader"
+import BackActionHeader from "./example/BackPreviousNextCloseActionHeader"
+import MaximizeActionHeader from "./example/MaximizeCloseActionHeader"
+import ActionHeaderAllergies from "./example/ActionHeaderAllergies"
+
+<Badge />
+
+# Accessibility Guide for Terra Action Header
+
+
+## Why is this important?
+
+> Terra Action Header is an essential component used to identify a section of content and provide controls to manage the respective content. If Terra Action Header is not made accessible, it can prevent users from understanding content and necessary context to interact with the page.  
```

Comment:
```suggestion
> Terra Action Header is an essential component used to identify a section of content and provide controls to manage the respective content. If Terra Action Header lacks accessibility, it can prevent users from understanding content and necessary context to interact with the page.  
```